packages/api/src/utils.ts

```diff
@@ -34,3 +35,48 @@ export const getQueryKeys = (name: string, props?: Record<string, unknown>) => {
 
     return [name, query_props];
 };
+
+const AUTH_REQUIRED_ENDPOINTS = [
+    'account_security',
+    'balance',
+    'get_account_status',
+    'get_account_types',
+    'get_available_accounts_to_transfer',
+    'get_financial_assessment',
+    'get_limits',
+    'get_self_exclusion',
+    'get_settings',
+    'mt5_login_list',
+    'new_account_real',
+    'new_account_virtual',
+    'new_account_wallet',
+    'p2p_order_list',
+    'payout_currencies',
+    'portfolio',
+    'proposal_open_contract',
+    'statement',
+    'topup_virtual',
+    'trading_platform_accounts',
+    'trading_platform_available_accounts',
+    'trading_servers',
+    'transaction',
+    'transfer_between_accounts',
+    'wallet_migration',
+];
+
+export const isAuthorized = (name: TSocketEndpointNames) => {
+    let is_authorized = true;
```

Comment:
why `is_authorized` is `true`, shouldn't it be `false` as a default for authorization?

Reply:
@shahzaib-deriv it's true because it can be request which doesn't need to be checked for authorize. We check it later